src/metabase/util/ssh.clj

```diff
@@ -0,0 +1,95 @@
+(ns metabase.util.ssh
+  (:require [clojure.tools.logging :as log]
+            [metabase.util :as u])
+  (:import com.jcraft.jsch.JSch))
+
+(def default-ssh-timeout 30000)
+
+(defn start-ssh-tunnel
+  [{:keys [tunnel-host tunnel-port tunnel-user tunnel-pass host port]}]
+  (let [connection (doto (.getSession (new com.jcraft.jsch.JSch) tunnel-user tunnel-host tunnel-port)
+                     (.setPassword tunnel-pass)
+                     (.setConfig "StrictHostKeyChecking" "no")
+                     (.connect default-ssh-timeout)
+                     (.setPortForwardingL 0 host port))
+        input-port (some-> (.getPortForwardingL connection)
+                           first
+                           (clojure.string/split #":")
```

Comment:
You should definitely make sure to `require` `clojure.string` and use the alias. Otherwise we're implicitly dependent on some other namespace requiring `clojure.string` in order for things to work properly, which is sort of a bad habit to get into

```clojure
(:require [clojure.string :as str])

(str/split ...)
```